server/database/models/user.js

```diff
@@ -3,11 +3,11 @@ export default (sequelize, DataTypes) => {
     {
       firstname: {
         type: DataTypes.STRING,
-        allowNull: false
+        allowNull: true
```

Comment:
@shaolinmkz, is there a reason you set the last name and first name to allow Null.

Reply:
It was due to the Twitter social login, Twitter doesn't return `firstname` and `lastname` just `display name` and `username`. So if I make it required it returns an error when data from twitter is being saved to the database.